Tests/Functional/Mailer/Transport/SparkpostTransportTest.php

```diff
@@ -127,7 +140,7 @@ private function assertSparkpostTestRequestBody(string $body): void
         Assert::assertSame('Hi! This is a test email from Mautic. Testing...testing...1...2...3!', $bodyArray['content']['text']);
     }
 
-    private function assertSparkpostRequestBody(string $body): void
+    private function assertSparkpostRequestBody(string $body, bool $expectedTrackingConfig): void
```

Comment:
The parameter `$expectedTrackingConfig` is not used.

Reply:
Great catch! It turns out the assertions were muted by a code that was catching generic `Exception`. There were already several assertions failing but the tests were green. Fixed it.